docs/application/dotnet/guides/internationalization/localization.md

```diff
@@ -0,0 +1,253 @@
+        {
+            CultureInfo info = GetCurrentCultureInfo();
+            _currentCultureInfo = info;
+            // Notify the change of locale information
+            MessagingCenter.Send<ILocalize, CultureInfo>(this, "LanguageChanged", info);
+        }
+
+        string TizenToDotnetLanguage(string tizenLanguage)
+        {
+            var netLanguage = tizenLanguage;
+            //certain languages need to be converted to CultureInfo equivalent
+            switch (tizenLanguage)
+            {
+                case "zh-CN":   // Chinese Simplified (People's Republic of China)
+                    netLanguage = "zh-Hans"; // correct code for .NET
+                    break;
+                case "zh-HK":  // Chinese Traditional (Hong Kong)
+                case "zh-hk":
+                case "zh-tw":  // Chinese Traditional (Taiwan)
+                case "zh-TW":
+                    netLanguage = "zh-Hant"; // correct code for .NET
+                    break;
+            }
+
+            Console.WriteLine("[Speedmeter] .NET Language/Locale:" + netLanguage);
+            return netLanguage;
+        }
+    }
+}
+
+```
+
+- Third. use the DependencyService in the Xamarin.Forms App to call the interface and set your `RESX` resources culture to the correct value.
+  you can receive culture info using `MessagingCenter` and update your app UI when your app is running and system Language has been changed.
+
+```cs
+var ci = DependencyService.Get<ILocalize>().CurrentCultureInfo;
+Resx.AppResources.Culture = ci; // set the RESX for resource localization
+DependencyService.Get<ILocalize>().SetLocale(ci);
+
+// Whenever language has been changed, CurrentCulture will be updated.
+MessagingCenter.Subscribe<ILocalize, CultureInfo>(this, "LanguageChanged", (obj, culture) =>
+{
+    Resx.AppResources.Culture = culture; 
+    DependencyService.Get<ILocalize>().SetLocale(culture);
+    //update your App UI
+});
+```
+
+
+<br><br>
+## Image Localization
+Tizen project supports localized images(resources) using different resource directories.
+Tizen uses res.xml file to specify information about the directory where localized resources(e.g. Image, Sound, and so on) are contained.
+The res.xml file is automatically generated when building your application.
+
+<br><br>
+**To add the localized resource directories in your application in Visual Studio:**
+
+- In Solution Explorer, Select Tizen project. And then, Go to Tools -> Tizen -> Resource Manager<br>
+![local_res_mgr_1](media/local_res_mgr_1.png)
+
+<br><br>
+- Resource Manager opens configurations -> Open the language drop-down list<br>
+![local_res_mgr_2](media/local_res_mgr_2.png)
+
+<br><br>
+- Select the language to add -> click Add button<br>
+![local_res_mgr_3](media/local_res_mgr_3.png)<br>
+![local_res_mgr_4](media/local_res_mgr_4.png)
+
+<br><br>
+- Resource directories are automatically created in Tizen project as follows.<br>
```

Comment:
Resource directories are automatically created in Tizen project as:

Reply:
done